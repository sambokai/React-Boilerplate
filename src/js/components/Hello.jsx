import React from 'react';

class Hello extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      name: 'Sam',
    };
  }

  render() {
    return (
      <h3>
        Hello, {this.state.name}! I am React.
      </h3>
    );
  }
}

export default Hello;
