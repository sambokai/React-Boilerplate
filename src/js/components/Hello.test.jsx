import React from 'react';
import { shallow } from 'enzyme';

import Hello from './Hello';


it('should render correctly', () => {
  const wrapper = shallow(<Hello />);

  expect(wrapper).toMatchSnapshot();
});
