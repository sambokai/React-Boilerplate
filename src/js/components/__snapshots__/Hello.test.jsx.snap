// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<h3>
  Hello, 
  Sam
  ! I am React.
</h3>
`;
